docs/ocis/identity-provider/oidc/flow.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Flow
+weight: 40
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/ocis/identity-provider/oidc
+geekdocFilePath: flow.md
+---
+
+In Infinite Scale, authentication can follow one of the three methods described on the [official site](https://openid.net/specs/openid-connect-core-1_0.html#rfc.section.3):
+1. Authorization Code Flow
+2. Implicit Flow
+3. Hybrid Flow
+
```

Comment:
From the list of three items above:
* is this one needs configuration in ocis to be set,
* or ocis is capable of doing these tree and eg the IPD needs configuration settings.

Needs a clarification text to make things clear.